agrirouter-sdk-dotnet-standard-impl/Service/Onboard/RevokeService.cs

```diff
@@ -37,7 +37,7 @@ public RevokeService(Environment environment, HttpClient httpClient)
         /// </summary>
         /// <param name="revokeParameters">The parameters for the revoke process.</param>
         /// <param name="privateKey">The private key.</param>
-        /// <exception cref="RevokeException">Will be thrown if the revoking was not successful.</exception>
+        /// <exception cref="OnboardException">Will be thrown if the revoking was not successful.</exception>
```

Comment:
I do not like the concept of changing the specialized business exception to a more generic `OnboardException`. I would suggest having a common base class and keep the specialized exceptions.

Reply:
I created `OnboardRevokeExceptionBase` (any ideas about a better name?) base exception and inherited `OnboardException` and `RevokeException` from it.